### PR TITLE
Changing the 1.22.0-beta.1 release date to July8

### PR DESCRIPTION
**Pull request by savitharaghunathan**
#### What type of PR is this:

/kind cleanup
/kind documentation

#### What this PR does / why we need it:
Correcting 1.22.0-beta.1 release date to reflect the planned date

#### Which issue(s) this PR fixes:
None

#### Special notes for your reviewer:
None

/cc @kubernetes/release-team-leads 
/assign @puerco

### CODE REVIEW

File: releases/release-1.22/README.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ The 1.22 release cycle is proposed as follows:
 | **Call for [Exceptions][Exception]** | Lead | Mon June 28 | week 10 | |
 | Brace Yourself, Code Freeze is Coming | Comms / Bug Triage | Mon June 28 | week 10 | |
 | **Begin Feature blog freeze** (23:59 PDT) | Comms Lead | Mon June 28 | week 10 | |
-| 1.22.0-beta.1 released | Branch Manager (TBD) | Thu June 8 | week 11 | |
+| 1.22.0-beta.1 released | Branch Manager (TBD) | Thur July 8 | week 11 | |
```
Comment (saschagrunert) [inline releases/release-1.22/README.md:65]:
```suggestion
| 1.22.0-beta.1 released | Branch Manager (TBD) | Thu July 8 | week 11 | |
```

**Review by saschagrunert — APPROVED** [review]

File: releases/release-1.22/README.md (hunk 1 above)
Comment (savitharaghunathan) [reply releases/release-1.22/README.md:65]: on it

**Comment by puerco** [comment]
Oops thanks for catching that @savitharaghunathan

**Review by saschagrunert — APPROVED** [review]

File: releases/release-1.22/README.md (hunk 1 above)
Comment (hasheddan) [reply releases/release-1.22/README.md]: @saschagrunert is it intentional that `beta.2` comes just 5 days later?

Comment (puerco) [reply releases/release-1.22/README.md]: Yes we stuck that one in there intentionally to address a request from the community to have another beta after code freeze. But dates were tight and some were already fixed.

**Pull request merged**